app/assets/stylesheets/header-footer-only.scss

```diff
@@ -3,6 +3,7 @@ $govuk-use-legacy-palette: false;
 
 @import "govuk_publishing_components/govuk_frontend_support";
 @import "govuk_publishing_components/components/_cookie-banner";
+@import "govuk_publishing_components/components/_action-link";
```

Comment:
Do we need to add this to the error page css too?

Reply:
Yes, have added. Also noticed that `static.scss` imports this file, so have removed some duplication from it. I've done this as an extra commit, can you sanity check please?